fastlane/Fastfile

```diff
@@ -207,15 +226,28 @@ UPLOAD_TO_PLAY_STORE_JSON_KEY = File.join(PROJECT_ROOT_FOLDER, '.configure-files
     )
 
     download_translations(options)
-    android_bump_version_final_release()
+    android_bump_version_final_release
 
-    version = android_get_release_version()
-    download_metadata_strings(version: version["name"], build_number: version["code"])
+    version = android_get_release_version
+    download_metadata_strings(
+      version: version['name'],
+      build_number: version['code']
+    )
 
     # Wrap up
-    removebranchprotection(repository:GHHELPER_REPO, branch: "release/#{version["name"]}")
-    setfrozentag(repository:GHHELPER_REPO, milestone: version["name"], freeze: false)
-    close_milestone(repository:GHHELPER_REPO, milestone: version["name"])
+    removebranchprotection(
```

Comment:
💡  Renamed `remove_branch_protection` in release-toolkit `9.1.0`; to keep in mind to address in future PR.

Reply:
Fixed in https://github.com/Automattic/simplenote-android/commit/5be41c297a5bc132c30f082f25f623e5fd4c9860. I went ahead and updated `remove_branch_protection` and replaced `setbranchprotection` with `copy_branch_protection`.